contents/blog/duolingo-testing-lessons.md

```diff
@@ -0,0 +1,107 @@
+
+![Super Mario Bros. Visual](https://res.cloudinary.com/dmukukwp6/image/upload/mario_onboarding_4d694891db.jpg)
+
+The first level of Super Mario Bros. is considered a masterclass in onboarding. There is no expository pop-up that says "Welcome! Press the right arrow button to start moving right." Instead, you start situated at the left of the screen; so it's intuitively obvious that you should probably start moving to the right, where Mario is pointing and all the stuff seems to be located. We can take onboarding cues from examples like this and Duolingo to decrease friction in the onboarding experiences we build.
+
+## 🎯 Lesson 4: Help people with _their_ goals (not yours)
+
+If word of mouth is your primary marketing vehicle, you want people to tell their friends about how awesome your product made _them_, not just how awesome your product is. Consider the following:
+
+> "I just got back from my dream vacation to Venice!"
+> "Really? Do you speak Italian?"
+> "I didn't have time to take a long course. But I did a couple Duolingo lessons every day for the past few months. By the time I got to Venice, I had no problem getting by."
+
+For Duolingo, it's important that users keep coming back. Not only because it's good for their company, but also because that's how learning works. It's hard to learn a new language if you're only studying once a week. If you can build a consistent study habit, you can chip away at it and learn over time. It's a huge opportunity for Duolingo to make their product more engaging, but also to help users achieve their goals of learning a language.
+
+When it comes to product engagement, there is a fine line between:
+
+- spamming people with reminders to get them to keep using your product
+- encouraging using your product because it's genuinely helping them achieve a goal
+
+A/B testing streak reminders provides a great example of aligning these goals. They tinkered with the copy, the timing, the frequency, and personalization of these reminders to get them just right. If you think about it, it doesn't make sense to get a streak reminder while you're asleep. And users probably have different times of the day when they are able to allocate time for study. Running experiments allowed them to find compounding improvements for personalized copy at just the right time of day.
+
+## 🏁 Conclusion
+
+Here are a few key takeaways to think about when building your product:
+
+- Think of your product as a game where you can run A/B tests to constantly tweak details.
+- Add a generous free tier to encourage more people to try your product
+- Use tests to decrease friction in the onboarding experience
+- Balance engagement intentionally by aligning with user goals
```

Comment:
This conclusion falls a little flat for me – I have no idea "Balance engagement intentionally by aligning with user goals" really means. 

Rather than repeating the points, could you get a bit more explicit and come up with something like "Ideas you can try" and come up with some hypothetical examples that embody the lessons. One or two may be enough, it just needs a tiny bit more meat to make it feel satisfying.

Reply:
Agreed. I replaced the conclusion section with a short list of actionable ideas:
https://github.com/PostHog/posthog.com/pull/9423/commits/5a3ae093e0d60df2ba0670a5e851886768c4df64